src/borg/platform/freebsd.pyx

```diff
@@ -50,59 +50,77 @@ cdef extern from "unistd.h":
     int _PC_ACL_NFS4
 
 
-def listxattr(path, *, follow_symlinks=False):
-    ns, prefix = EXTATTR_NAMESPACE_USER, b'user.'
+# On FreeBSD, borg currently only deals with the USER namespace as it is unclear
+# whether (and if so, how exactly) it should deal with the SYSTEM namespace.
```

Comment:
Support for SYSTEM would need a separate PR from an experienced FreeBSD user/developer.

The scope of this PR here is to make the current borg behaviour less broken, especially when restoring archives that were made on other platforms onto a FreeBSD system (they could have xattr names with other namespaces than "user").